### PR TITLE
INC-22/1-Add-Profile-Photo

### PR DESCRIPTION
**Pull request by maskpa100**

### CODE REVIEW

File: assets/api/instance.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const token = getAccessTokenFromCookie()
 
 //to get client-side requests
 export const instance = axios.create({
-  baseURL: 'https://inctagram.work/api/',
+  baseURL: 'https://inctagram.work/api/v1',
```
Comment (TURP1) [inline assets/api/instance.ts:10]: https://inctagram.net/api/v1

File: assets/locales/ru/translationRu.json [hunk 2]
```diff
@@ -98,5 +98,12 @@
     "save_changes": "Сохранить изменения",
     "modal_error_title": "Извините!",
     "modal_error_content": "Ваши данные не были изменены. Пожалуйста, перезагрузите страницу."
+  },
+  "add_profile_photo": {
+    "add_profile_photo": "Добавить фотографию",
+    "select_from_computer": "Выберать с компьютера",
```
Comment (TURP1) [inline assets/locales/ru/translationRu.json:104]: Выбрать

File: assets/locales/en/translationEn.json
Comment (TURP1) [inline assets/locales/en/translationEn.json:1]: esLint check

**Review by TURP1 — APPROVED** [review]

**Comment by TURP1** [comment]
resolve the conflicts on your's branch

File: components/profile/AddAvatar/ModalAvatar/Start/start.module.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    img {
+      color: #fff;
+    }
+  }
+
+  .button {
+    margin-top: 45px;
+
+    button {}
```
Comment (Viarcinski) [inline components/profile/AddAvatar/ModalAvatar/Start/start.module.scss:28]: Better remove it I guess

File: assets/api/instance.ts (hunk 1 above)
Comment (Viarcinski) [inline assets/api/instance.ts:10]: Better replace it with "baseUrl" from common.api.ts

**Review by Viarcinski — APPROVED** [review]
Nice work!

File: assets/api/user/profileQueryApi.ts [hunk 4]
```diff
@@ -33,9 +33,28 @@ export const profileQueryApi = createApi({
         body: userData,
       }),
     }),
+    avatarAdd: builder.mutation({
```
Comment (TURP1) [inline assets/api/user/profileQueryApi.ts:36]: add TS

File: assets/api/user/profileQueryApi.ts [hunk 5]
```diff
@@ -33,9 +33,28 @@ export const profileQueryApi = createApi({
...
+      query: (file) => ({
+        url: userRouts.avatar,
+        method: 'POST',
+        body: file,
+        headers: {
+          Authorization: `Bearer ${localStorage.getItem('accessToken')}`,
+        },
+      }),
+    }),
+    avatarDelete: builder.mutation({
```
Comment (TURP1) [inline assets/api/user/profileQueryApi.ts:46]: add TS

File: package.json
Comment (TURP1) [inline package.json:1]: still problems with lint

**Review by ilich01 — APPROVED** [review]

**Review by FightersForJustice — APPROVED** [review]

**Review by ilich01 — APPROVED** [review]

**Review by irinrudko — APPROVED** [review]

**Pull request merged**